railties/lib/rails/generators/rails/app/templates/config/environments/test.rb.tt

```diff
@@ -12,8 +12,8 @@ Rails.application.configure do
   config.enable_reloading = false
 
   # Eager loading loads your whole application. When running a single test locally,
-  # this probably isn't necessary. It's a good idea to do in a continuous integration
-  # system, or in some way before deploying your code.
+  # this probably isn't necessary. It's a good idea to enable it in a continuous integration
+  # system, or in *some* way, before deploying your code.
```

Comment:
Let's make this less casual overall. I'm thinking something like the following would be more informative.

"Eager loading loads your entire application. When running a single test locally, this is usually not necessary. It is recommended that you enable it in continuous integration systems to ensure eager loading is working properly before deploying your code."

Reply:
:+1: made a few tweaks, but updated nonetheless